test_scripts/Polices/App_Permissions/ATF_HP_User_Consent_NO.lua

```diff
@@ -0,0 +1,309 @@
+---------------------------------------------------------------------------------------------
+-- Description:
+-- On getting negative user consent from HMI and storage this data into LocalPolicyTable, SDL must:
+-- 1. assign only the permissions for which there is user consent in the local policy table and permissions (if any) that are listed in 'pre_DataConsent' section (that is, 'default_hmi', 'groups' and other) for this application 
+-- 2. notify HMI via onPermissionsChange() notification about the applied permissions.
+-- 3. notify mobile app via onPermissionsChange() notification about the applied permissions.
+-- Preconditions:
+-- 1. Delete policy.sqlite file, app_info.dat files
+-- 2. <Device> is connected to SDL and consented by the User, <App> is running on that device.
+-- 3. <App> is registered with SDL and is present in HMI list of registered aps.
+-- 4. Local PT has permissions for <App> that require User`s consent.
+--
+-- Requirement summary:
```

Comment:
This section should be first please move it to top. 
Also please add here only name of requirement. In your case it should look like:
-- Requirement summary:
-- [Policies]: User-consent "NO"